Scripts/Linux/RunFunctionalTests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Let's skip this script for now. Bring it back when we have builds targeting it.

Reply:
My only caveat to that is that I use it to run the functional tests manually, to see what's working and what's not.  So it's quite useful even if it's not being run by a CI job yet.